teapot/cogs/nqn.py

```diff
@@ -0,0 +1,100 @@
+import discord
+from discord import utils
+from discord.ext import commands
+
+
+class Emoji(commands.Cog):
+	def __init__(self, bot):
+		self.bot = bot
+
+	async def getemote(self, arg):
+		emoji = utils.get(self.bot.emojis, name = arg.strip(":"))
+
+		if emoji is not None:
+			if emoji.animated:
+				add = "a"
+			else:
+				add = ""
+			return f"<{add}:{emoji.name}:{emoji.id}>"
+		else:
+			return None
+
+	async def getinstr(self, content):
+		ret = []
+
+		spc = content.split(" ")
+		cnt = content.split(":")
+
+		if len(cnt) > 1:
```

Comment:
```suggestion
		if len(cnt) > 1:
					for item in spc:
				if item.count(":") > 1:
					wr = ""
					if item.startswith("<") and item.endswith(">"):
						ret.append(item)
					else:
						cnt = 0
						for i in item:
							if cnt == 2:
								aaa = wr.replace(" ", "")
								ret.append(aaa)
								wr = ""
								cnt = 0

							if i != ":":
								wr += i
							else:
								if wr == "" or cnt == 1:
									wr += " : "
									cnt += 1
								else:
									aaa = wr.replace(" ", "")
									ret.append(aaa)
									wr = ":"
									cnt = 1

						aaa = wr.replace(" ", "")
						ret.append(aaa)
				else:
					ret.append(item)
		else:
			return content

		return ret
```
Please read [PEP 8 -- Style Guide for Python Code](https://www.python.org/dev/peps/pep-0008/#names-to-avoid)